iscservice/demo/iscdemo.c

```diff
@@ -0,0 +1,607 @@
+/* -*- C -*- */
+/*
+ * Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For any questions about this software or licensing,
+ * please email opensource@seagate.com or cortx-questions@seagate.com.
+ */
+
+#include <unistd.h>       /* getopt */
+
+#include "lib/memory.h"   /* m0_alloc m0_free */
+#include "rpc/conn.h"     /* m0_rpc_conn_addr */
+#include "rpc/session.h"  /* iop_session */
+#include "xcode/xcode.h"
+#include "conf/schema.h"  /* M0_CST_ISCS */
+#include "layout/layout.h" /* M0_DEFAULT_LAYOUT_ID */
+#include "layout/plan.h"  /* m0_layout_plan_build */
+#include "motr/client.h"
```

Comment:
"motr/client.h" is, and should be the only header file a user application needs to include.
All other header files are included by "motr/client.h".

Reply:
I'm not sure. Not all apps need to use layout/plan, for example...

Reply:
OK, that's fine.